pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Participating Group Members:
```

Comment:
PR template looks great. I really like how there are multiple areas to communicate our approaches. Will be especially helpful managing a group of this size & communicating between FE & BE

Reply:
I completely agree!